examples/undocumented/python/classifier_multiclassocas.py

```diff
@@ -4,7 +4,6 @@
 
 def classifier_multiclassocas (num_vec=10,num_class=3,distance=15,width=2.1,C=1,epsilon=1e-5,seed=1):
 	from shogun import MulticlassLabels
-	from shogun import Math_init_random
```

Comment:
you actually don't need to replace the seed calls in the undocumented examples -- those were only important for integration tests (the old ones) but we dont have them anymore ... just remove the init_seed calls everywhere and all good

Reply:
although no need to revert the added .put(seed) calls